.github/workflows/md5_check_initial-data.yml

```diff
@@ -21,4 +21,4 @@ jobs:
         run: md5sum docker/initial-data.json
 
       - name: Validate and protect initial-data.json
-        run: echo "f6274a3f6f638f55a4c66f963d3abbca  docker/initial-data.json" | md5sum -c -
+        run: echo "99fc335aa19a73085432c0e45d417856  docker/initial-data.json" | md5sum -c -
```

Comment:
Not for this PR, but: If we change the initial data with every change to the models, this job does not make sense. It was intended to prevent accidental changes to the file, but if the file gets changed too often, this is just useless work.